Functions/Quantification/xASL_quant_HadamardDecoding.m

```diff
@@ -177,6 +166,13 @@
 	error('Length of Initial_PLD vector does not match the number of volumes.');
 end
 
+% Prepare the parameters also for interleaved reordering
+nRepetitions = nEncodedVolumes / (xQ.TimeEncodedMatrixSize * xQ.NumberEchoTimes); % Calculating no. of acquisition repeats accounting for multipleTE
```

Comment:
If you don't need matrix multiplications/divisions, do you mind adding a . dot/period in front of it, so when I read it, I understand that this is a normal division/multiplication and not a matrix multiplication/division? Because without knowing the dimensions of a variable, I don't know that. It would help me understand it. Thanks!

Reply:
Good point.